Yoast/wordpress-seo
# Show score text in the posts and taxonomies lists mobile view.

## Summary

This PR can be summarized in the following changelog entry:

* Show the SEO and Readability score text in the posts and taxonomies list mobile view.

## Relevant technical choices:

- makes the score screen-reader-text visible in the mobile view

## Test instructions
- check the posts, pages, and taxonomies overview pages in the WP admin
- in the mobile view, check the score text is displayed on the right of the score icons

Examples:

<img width="751" alt="screenshot 2018-12-14 at 11 35 05" src="https://user-images.githubusercontent.com/1682452/50000236-317aa300-ff99-11e8-860d-a0ac1d026117.png">

<img width="691" alt="screenshot 2018-12-14 at 11 56 49" src="https://user-images.githubusercontent.com/1682452/50000237-317aa300-ff99-11e8-90aa-8ed5fe164d12.png">


Fixes #7654
